src/Framework/Database/DB.php

```diff
@@ -81,6 +82,17 @@ function () use ( $name, $arguments ) {
 		);
 	}
 
+	/**
+	 * Get last insert ID
+	 *
+	 * @since 2.10.0
+	 * @return int
+	 */
+	public static function last_insert_id() {
```

Comment:
@ravinderk yes, but then we should map all existing `wpdb` methods like `get_result`, `get_var`, etc. to have consistent API. In the future, this class will probably be updated a lot, it will be a good idea to think more about what we want from this class and not just update the method name. For now, I think we should leave it as is to match the `wpdb` API.

Reply:
@alaca I agree with you but we have to follow a naming pattern `src` directory and this function name does not follow this.

cc: @JasonTheAdams @kjohnson

Reply:
@alaca I agree with you.